### ISSUE TITLE
Rewrite notebooks under tests/ folder

### ISSUE THREAD

**Issue by queirozfcom** [scikit-multilearn/scikit-multilearn#149]
The notebooks under `tests/` were outdated, calling old APIs and package names.

- **BRkNNTesting.ipynb**: rewritten using the new classes.

- **CrossValTesting:**: I didn't quite understand how you were classifying the RCV1 dataset without first performing feature extraction for text, tf-idf and so on.. Since the objective is just to show how cross-validation testing, I've used a smaller dataset instead (scene). Also added a couple of metrics.

- **MLkNNTesting.ipynb:** I rewrote everything but the match between the Mulan MLkNN and the skmultilearn MLkNN was not 100%.

- **RakelDTesting.ipynb**. I rewrote everything and, again, the match between Mulan VS skmultileran results isn't good. Maybe the Mulan algorithm version is different?

**P.S.:** Some results are different from those that were in these notebooks before. Do you want to have a look and see if they're correct? @niedakh 

**P.S. 2:** These are more like examples, right? Should we perhaps rename `tests/` to `examples/` and keep `tests/` for actual unit tests?

**Comment by ChristianSch** [comment]
We used to test everything by running the notebooks. This was superseded by actual unit tests. So I guess renaming this to `examples` would make sense. They should then be refactored to showcase different approaches, such as the cross validation, instead of trying to validate some aspects of the library. I don't really know how the state of unit testing is though.

In general I feel like a more detailed approach to committing would be preferable. Changing the files ignored by gitignore in a commit with the message with "fixing $issue" is bad for accountability in the future. If you would've changed only the imports and such, making the notebooks compatible with the most recent version of skmultilearn, we could compare the performance better I guess. Same for more detailed changes you made.

I'm not really sure we need those notebooks anyway in the future. @niedakh could chime in on this.

Relevant discussion: https://github.com/scikit-multilearn/scikit-multilearn/issues/73

**Comment by queirozfcom** [comment]
@ChristianSch you are right about the gitignore file being edited here. I overlooked it. Shall I make another PR?

**Comment by ChristianSch** [comment]
As I said, I think it's preferable to keep the overall changes to a minimum. I don't think rewriting everything helps. Personally, I would prefer a PR with only API changes being fixed, without rewriting all the notebooks.
Let's wait for niedakh anyway.
